### ISSUE TITLE
Use operational emissions instead of lifecycle emissions

### ISSUE THREAD

**Issue by HansHyde** [tmrowco/electricitymap#960]
Just noting #926 "Moldova" discussion & the Type_Mapping for generators...

Would it be possible to create a sheet/file similar (possibly not "functional" at this time) to CO2eq_parameters.js, where we define power plant types (particularly thermal) by characteristics?  This could then be used as EM calculation methodology support (with required supporting documentation, not just _"source - 'Hans Hyde'"_) to back EM's CO2eq x kWh data aggregations.

These "energy conversion" characteristics of the power plant ultimately determine how much CO2 is produced per kWh by fuel type.  This is referred to as both "Heat Rate" and its mathematical inverse "Efficiency", and the point at which HR is optimal is under full, static power production --> highest kWh per unit energy input.  

Starting or Stopping, referred to as "ramping" returns an amount of CO2 without usable kWh produced based on the amount of fuel consumed during "ramping Up" & "ramping Down".  

- For so called "peakers", ramping occurs twice a day (almost like clockwork - and it has been happening this way for decades.
- Existing Pumped Storage does not exist today as a "pre-thought" to future intermittent renewable energy sources, it exists because of the "Nighttime Trough" - too much non-rampable generators (coal/nuclear) out producing the demand during the NT.  Without PS, transmission grids would burn down when generation far-exceeded load.

![daily_load_curve-typical](https://user-images.githubusercontent.com/33690988/34454003-771f9730-ed61-11e7-8461-5566f0fec963.jpg)
_Typical Daily Load (Generation) Curve - since the dawn of bulk electric power systems_

For anything using Steam as the prime mover [of the electric generator], this CO2 value is very high per fuel [excluding nuclear] and takes many hours to "ramp".  I honestly think we could capture these events within the backend coding from the realtime data streams.

A general observation, not specific to EM, is that plant types (if listed as abbreviations, or in different languages) are frequently misinterpreted by almost all in the "energy & electric power industry" - except for those that actually work with thermal generators themselves.  The IPCC 2014 Synthesis Report, Chapter 7 - Energy Systems (where CO2eq_parameters.js derives its values) IMHO steamsrolls over this, assigns a range almost generically based on fuel type (which works for low carbon generators to capture a "lifecycle value", but falls apart for thermal).  

![heatrate_co2emissions](https://user-images.githubusercontent.com/33690988/34454018-c793bfde-ed61-11e7-8c63-1d759d47125e.png)
_Heat Rate Impacts on CO2 Emissions per Prime Mover & Fuel Type_

[HeatRate_CO2emissions.xlsx.pdf](https://github.com/tmrowco/electricitymap/files/1594350/HeatRate_CO2emissions.xlsx.pdf)

Our data then becomes an increasing aggregation of datapoints equal to reported **kWh** (in the BPS bulk power system) x **Fuel Type** (assigned from sweeping averages in the IPCC 2014) = CO2eq (quantity).

I'm reading more information regarding the entities we are obtaining the our RTD streams from, **not reporting not only "ramping"** - as they cannot without the unit synchronized to the grid, **but also not reporting "spinning reserve" operations**.  For example, generation is 10,000 MW (CO2/kWh is captured in EM) and Spinning Reserve is roughly 4% - 400 MW (CO2/kWh **is not captured** in EM as RTD does not report it)

Contracted "out of Balancing Area" generation appears to follow similar patterns.  For example, a 500 MW coal generator may be within geography A (Pennsylvania, NJ, Ohio, etc) and is contracted to export to geography B (NY state).  Balancing Area A (PJM) does not record it within its Generation Mix RTD, _as it is producing for BA B_, and Balancing Area B (NYISO) does not record it within its Generation Mix RTD, _as it is producing in BA A_.

In all these regards we are missing CO2 (kG) actually emitted and part of the CO2eq/kWh data compiled.

I think we can use this to refine the data aggregated as well as standardize data reported per country (without the need for "exceptions" as is the case with Estonia) and uniformly cleanup the prevalence of "unknowns" or misclassification of "fuel type".

![eia 2016 electric gen by fuel type](https://user-images.githubusercontent.com/33690988/34454313-18f54f1c-ed69-11e7-9878-151e1c3328ac.png)
_EIA 2016 Gen by Fuel Type - (I'm working on integrating "Heat Rates Impacts" into it)_

As I noted to @jarek in #166 "add Canada" - Statistics Canada (as is EIA for the US) are good resources for "baseline" assumptions on the macro level, but better information is available at the ISO/RTO/System Operator Level, and it should be deferred to.  StatsCanada & EIA compile CO2 (other emissions) & actual generation numbers via monthly reporting of the each generator (which includes all emissions, ramping, spinning reserves, contracted generation, etc) - and actually allows for a check on the data EM is compiling.  We could potentially verify monthly CO2 produced per region as compiled by EM to the numbers reported by EIA/StatsCan to see how close/far off we are using IPCC's "average" values.

If we could tag this as data, science, backend, libraries it would be appreciated. ~hans

**Comment by corradio** [comment]
I think this would warrant several issues. If I understand correctly, here's a summary of issues you are raising:
1. Carbon intensity values should be broken down in further detail from (coal, gas, ...) to (anthracite, bituminous, ...) -- in that case we're discussing this in #162 
2. Carbon intensity values defined in co2eq_parameters.js could be customised per zone based on the type of thermal load installed, possibly taking into account the distribution of type of installed capacity, centralised in a sheet.
3. Spinning reserves are not captured the data we have
4. Contracted generation is not necessarily capture in the data we have (although for Brazil it actually already is done properly -- we count sold power as locally produced as we know there is a direct connection, and thus no dilution of the carbon intensity).

For 1 I'd refer to the associated issue.

2 is a tricky issue as we want to strike a certain balance between trust (through simplicity) and accuracy. If our models become too complicated (because we use too many specific emissions factors), it becomes too hard for newcomers to follow along on the Electricity Map. The second thing is that there's no point in spending too much sweat on things that will only have marginal impact. As a guiding principle I'd say a 10-20% margin of error in *relative* carbon intensity (between countries) is the maximum we'd want to tolerate. In that view we should adjust carbon intensities of entities that really are exceptions (Estonia for example) and that drastically change emissions.
It's more important for us to spend our energy on adding data for the world (and having our very simplified fuel source categories) than to go fish after the last 1% improvements (as long as we're inside the 10% error range). Furthermore, maintaining a file with that level of detail will be problematic.
Long story short: if we see >10% errors, let's fix them. Else, it's probably not worth it.

3 and 4 are quite complicated and in my opinion not worth discussing if we don't have any means of getting access to data.

**Comment by HansHyde** [comment]
Hey @corradio - I have some good news, and some bad news.

The goods... 

- **Natural Gas** was much more accurate (based on a data set for all energy generators in California for 2016) than I thought.  Only 7.61% variance between what EM would have calculated from kWh and then CO2 emissions expected from Primary Fuel Burn ONLY (no "CO2eq - lifecycle" including other factors; embedded CO2 plant, CO2 in fuel delivery, fugitive methane emissions, etc., which I believe IPCC 2014 is trying to capture.
- **Coal** was off by 27.4% (and I weighted for the 4 different types).  I found another set of CO2 content values for coal (slightly higher/worse), but used EIA values in the calculations.  However, 94.3% reported in the data set was from outside California in Utah at a plant owned by LADWP, so potentially No. 4 above is also an issue.

The bad... the kWh x IPCC g CO2eq/kWr calculation has significant issues...
- **Biomass** - off by 724%
- **Geothermal** - 100% of EM reporting is above & beyond US EIA or California Emissions Commission (CEC)
- **Hydro** - 100% of EM reporting is above & beyond US EIA or CEC
- **Nuclear** - 100% of EM reporting is above & beyond US EIA or CEC
- **Solar PV** -  - 100% of EM reporting is above & beyond US EIA or CEC
- **Wind** - 100% of EM reporting is above & beyond US EIA or CEC

Indifferent or Outlier Data
- **Solar Thermal** - a data set anomaly, as CO2 reported was for the natural gas backup unit, but measured against all MWh generated
- **Oil** - no generation in California

I'll comment more later, but have to get off the computer now.   Wish there was an easy answer... but I don't think this methodology is supportable. :(

![co2 power emissions - 2 methodologies comparied](https://user-images.githubusercontent.com/33690988/34463029-c3ea2e46-ee50-11e7-8741-bb30bda195f5.png)


[CO2 Power Emissions - 2 Methodologies Comparied.pdf](https://github.com/tmrowco/electricitymap/files/1595295/CO2.Power.Emissions.-.2.Methodologies.Comparied.pdf)

**Comment by HansHyde** [comment]
PS - I think the "Carbon Intensity in the last 24 hrs" needs a disclaimer or "Beta" stamp on it.

Here is the disclaimer PJM uses for its realtime data "Electricity Markets" ...

> Preliminary integrated hourly loads are calculated from raw telemetry data and are approximate.
> 
> This information is provided for informational purposes only and should not be relied upon by any party for the actual billing values.
> 
> Preliminary loads do not reflect confirmed transaction information with the market participants and the external control areas.

http://pjm.com/markets-and-operations/energy/real-time/hourly-prelim-loads.aspx

I was thinking something of this order needs to be applied to any parser/region that is not 100% accounted for around its borders.  NE-ISO & NY are good examples - as we don't have access to data from HydroQuebec, and right now I can guarantee with the "cold snap" happening, imports from HQ are nearly 100% of their interface capacity.

**Comment by HansHyde** [comment]
@brunolajoie @corradio where are we on this?

Do you not understand due to the complexity of the subject matter?  Or do you not like what I discovered because it hits at the very heart of the fundamental methodology in the backend of the ElectricityMap project itself?

I'm trying to work through a fix for this, knowing we really only have limited data available to work with, and as you are both aware I have discreetly (towards EM) sought the advice of experts (not only in energy itself, but also data sciences) to rectify this and bring us a workable solution.

Have you both read through the comments of my post on LinkedIn?  I did not realize Brice tagged you both yesterday, else I would have deleted it immediately.  Not because you should not know, but that I asked all not to mention by name EM, in the hopes you would both read the comments highlighting the problems with this methodology.  

https://www.linkedin.com/feed/update/urn:li:activity:6353613073135345664

There are too many conversations on here going back and forth on what gCO2/kWh rates should be, and modifying them for countries specifically.  You both return to using the IPCC's 2014 "lifecycle CO2 eq", but as the California dataset & my compiling of it shows, by the EM/IPCC calculation only natural gas's fuel burn (DIRECT CO2 Emissions) were captured accurately (but still off by 7.5%).

Burning of wood (biomass) releases the same amount of emissions as coal (its specific carbon content is roughly the same) in the realtime.  Yet we are calculating it at "230" versus "820" for coal?  CO2 in the atmosphere is CO2 in the atmosphere.  It doesn't matter whether it was locked away yesterday, 15 years ago or 5 million years ago... once released it is there.  So is EM calculating for realtime or "lifetime"?  Origin of the CO2 doesn't matter once it is in the atmosphere or even more so in the oceans.

I've spent morning to night the past 5 days trying to get us a workable calculation revolving around only have realtime bulk generation by very generic fuel type, to capture all of the following...   Extraction, Processing, Delivery/Transport, Transformation (Power/Heat generation), PP Embedded carbon, Transmission & Distribution... and base it all around 1 variable (gen output) used to make a "final" decision of what the emissions are.  It's not easy (nor simple), but it can't be ignored anymore.

**Comment by corradio** [comment]
Hi @HansHyde,
I've looked carefully at the analysis you made.
First of all, it seems like none of the points in my previous comments actually captured the issue you're raising (https://github.com/tmrowco/electricitymap/issues/960#issuecomment-354553094): the issue you're raising is whether we should adopt the **operational emissions** or **lifecycle emissions** methodology for calculating emissions. I have changed the title accordingly.

Neither of those approaches is "flawed" as you say, and both have their pros/cons. The important thing is to make clear which hypothesis is taken.

Operational emissions do not take into account the whole life cycle of the plant (construction, decommissioning, recycling, fuel transportation...). This is why wood/biomass for example is lower than coal using lifecycle emissions (if you re-plant trees). This is also why wind is not 0 (offshore wind requires boats running on oil to build turbines made of concrete). That CO2eq has to be accounted for in our opinion.
However, lifecycle emissions include an assumption on the lifetime of the asset studied.
As emission factors (CO2eq per kWh of generated electricity) are obtained by dividing lifetime emissions (in CO2eq) by the total electricity produced (in kWh) during its lifetime, there is a certain sensitivity on the precision of the estimated lifetime (which is acceptable in our opinions, and which is also why the IPCC has adopted this methodology).

As both approaches capture different things, it is expected that numbers differ significantly. We however believe, in the project, that lifecycle emissions capture the widest range of impact across the value chain, and thus we will continue with lifecycle emissions.

This project is still in its infancy but if we had more resources one could imagine having a switch between lifecycle and operational emissions in the frontend.

I hope this answers your questions.

**Comment by HansHyde** [comment]
@corradio 

From the FAQ in process... 

> +**Is non real time data useful?**  
> +While the map relies on having real-time data to work it's still useful to collect data from days/months past. This older data can used to show past emissions and build up a better dataset.

Strongly implies EM has the capacity to collect "real-time data" from the past, run it through the EM model, and see the results of CO2 emissions (lifetime or operational is not important) based on using the Generation Mix across time.

CAISO has datasets from the past available does it not?  Did you run them through the EM model and compare or preform a data analysis, against the dataset from California Emissions Commission for 2016 reported emissions against produced MWh?  I do not have the capacity, knowhow or accessibility to EM to do so.

Now let me be more direct...

IPCC 2014 Climate Synthesis Chapter 7 Energy provides "lifecycle gCO2eq per kWh" ranges per very generalized "fuel/plant" types.  It derives its **ranges** based on meta data analysis of various studies.  The **operational** data comes from monthly or yearly reporting of CO2 emissions and MWh produced (such as the CEC dataset), adjusted and corrected from actual (realized) data over a year.  These ranges are broad, and in no way represent data obtained from realtime calculations.

This is, for all practical purposes, peer-reviewed and I do not dispute it.  

What is peer-reviewed?  The ranges for "lifetime gCO2eq/kWh".

Now, you have through the EM project (which is amazing to be honest) derived a calculation to capture gCO2eq/kWh "lifetime" by multiplying against reported Real-time Data of kW generated with an incremental unit of time based on that reported in in the RTD.  

You have also set a fixed value for gCO2eq/kWh with pinpoint accuracy to 1 gCO2eq/kWh for each fuel type.  This is no longer a range, the accuracy of all data points generated by EM moving forward is exact.

And because we are talking in Megawatts & Gigawatts on a country basis, resulting in (for example, the US generated 4.08 Trillion KWhrs of electricity from all plants above 1MW installed capacity in 2016 as per US EIA), the resulting percision of every EM data point is...

0.000001 gCO2eq/MWh, or 
0.00000001 gCO2eq/GWh

This is not what IPCC 2014 is claiming, asserting or what has passed peer review.

The EM model is asserting this and effectively claiming precision of 1 significant value at 6th or 9th decimal place.

**The EM calculation model & critical assumptions are not peer-reviewed.  This does not matter whether it is "lifetime" or "operational".**

1. Has IPPC reviewed EM's model?
2. 
3. Has an ISO or a real-time data reporting agency reviewed EM's model?  
4. 
5. When @brunolajoie presented before ENSTO, did they give feedback or raise concerns over the methodology of the model when it comes to CO2eq production?
6. 
7. Have any power system engineers reviewed EM's calculation model?
8. 
9. Have any plant operators/engineers reviewed EM's calculation model?
10. 
11. Have any state or country level governmental agencies tracking CO2 emissions reviewed the EM calculation model?
12. 

Or is EM's model a really good idea implemented (realized) by yourself & Bruno based on an extremely simplistic metric reported across a range that seems appropriate to incorporate "1 for 1, as-is" into a program/product that aggregates reported real-time data that is one of the two data points needed to "balance" the equation?

Let's not forget, we "debated" for several hours over installed capacity of PV in NY to report in EM's generation capacity last week.  Against 1,200,000 MW of total installed capacity within the US of all generator types, +/- 200 MW represents 0.000167 precision & accuracy of...

from Bruno earlier today, 

> Thanks a lot for the insight alixunderplatz. Let's keep that in mind for when we will refine GHG-emission factors of German power plants.
> 
> As this PR only concerns installed capacities, I'm gonna merge it anyway, because capacities won't impact GHG-computation on the ElectricityMap.

Installed Capacity has nothing to do with any CO2 emissions, "operational" or "lifecycle", in EM's calculation.  This is actually something I am trying to configure, make usable within the EM calculator, as we are reporting/including Installed Capacity in the EM presentation.

So in conclusion, yes I agree the EM project is in its infancy, hence why I am trying to get a full spectrum working equation(s) & model based on the limited data we have available to us; 1) Generation Output by type" and 2) Installed Capacity by type.  And this is with knowing CO2 emissions per fuel type are fixed/specific based on energy content (unit measurable by electric generators).  

It is not as easy/simple as the equation used in the EM calculator, however it will be peer-reviewed and defensible.  It will also not claim precision to the 6th or 9th decimal, however it will be far more accurate in the real-time aggregated datasets.

The reach of the EM project's "data" is far including thought-leaders, persons of authority & decision makers.  It also includes those that will use the data for their own purposes, plotting one data set against another.  **So it is critical that the data & its calculations remain neural at its core.**

We can work together to improve this project (hopefully and my desire) or (and I really do not want to do this), I can compile data & calculations and present in a published report that critically examines the flaw in the methodology that is not peer-reviewed.

The first cannot happen, if yourself or Bruno are unable to recognize (or "see") that a fundamental flaw does exist in the EM model, not IPCC 2014's "ranges per fuel type" reporting.

Trying to resolve/explain/plan to resolve this is not happening through comments on "Issues", in the code edits, or on Slack.  And it is wasting far more time than either of us is having to spend on it - for no gain or even an agreement on a pathway towards resolution/improvement.

With respect, 

Hans

**Comment by corradio** [comment]
@HansHyde I am really really puzzled by your approach and reasoning.
Previously you were opposing operational emissions and lifecycle emissions (which is fine).
Now you are stating that we are incorrectly using IPCC values consisting of a median and a range (i.e. a statistical distribution).

> What is peer-reviewed? The ranges for "lifetime gCO2eq/kWh".

Well, the median values are too!

I don't see the problem of multiplying median carbon intensities with real-time production data. Do you? That ends up showing "most likely values" of the statistical distribution.
Now about the range: we are not saying **anything** about the range/uncertainty of calculations. We are therefore **not** assuming a precision of 1gCO2eq/kWh and the following conclusions you are drawing are not applicable.

After more than a million visitors, and many discussions with industry experts (consultants, TSOs etc..), none have been concluding that multiplying life-cycle median IPCC intensities with current real-time generation energies is a "flawed" calculation.

The "methodology" you are referring to is a multiplication of two numbers, and I don't see any reason to state that it is flawed at this point. Again, if you can point to reasonable arguments that clearly state that this multiplication is invalid (independently of the operational vs LCA approach), I'd be happy to take a look.

I do hope you understand that our time is valuable. In that interest, I hope you will make an effort to be concise on any subsequent issue you might have so we can have an efficient conversation.

**Comment by corradio** [comment]
I am closing the issue as I can't see any "flaw" in the way we use emission factors.
If anyone has any enlightening comments to put in light what I've failed to see, please re-open the issue (and I will be the first one to apologise).

**Comment by HansHyde** [comment]
Oli, @corradio, (fyi - @brunolajoie) 

You are correct in closing this "issue" as 

1. Neither Github nor Slack provide a fully-functional & efficient platform to define the problem and provide a pathway to a solution, 
2. It is wasting everyone's time (that is valuable), 
3. Information that is available (Installed Capacity) to Electricity Map is not incorporated into any calculations.  This means OCA realtime cannot be calculated (uniform variations across all fuel types) based on OA realtime data reporting when OCA values (recently closed or built, but never operated plants), are removed/never included within the Installed Capacity metric.

The analysis of data from California Emissions Commission 2016 reporting, was of OA (emissions versus produced MW) reported for a year - capturing direct Operational fuel burn only.  It was compared against emissions from an increment of time (1 year) that would be expected from Electricity Map's OCA calculations.

Summary of results, "Gas" OA,
1. Represented 55% of all Generation Output in the data set, and
2. Revealed a 7.6% variation from the emissions expected from a 1-year OCA calculation, 
3. Means 7.6% represents either a) a modeling/reporting error, or b) CO2eq emissions (OCA) for everything other than direct fuel burn.  You cannot accept both as correct explanations.

In Issue #973, I demonstrated 2 things;
1. the variance of OA for "Gas" was 1.95x - that is, the emissions of CO2 actual per reported kWh could be either 1 per kWh or nearly double, 2 per kWh,
2. a conversion factor regularly used for determining CO2 emissions from a fuel source is frequently misinterpreted, resulting in a margin of error of 3600x (60s x 60m).  The energy fuel input is not instantaneous, although the conversion does allow for obtaining an instantaneous (1 second) output.  The fuel input is over 1 hour, "per hour", so 1 second emissions are calculated based on total fuel input/3600 seconds.

If these do not raise concerns with you, over using OA data-only in a grossly simplified OCA methodology calculation, then at present, this is something I cannot hope to change efficiently or effectively.

I will stop wasting our time on Github/Slack, focus on preparing a more effective presentation (content & format), as well as return with a potential route to rectifying the degree of discrepancies.

Cheers,

Hans

**Comment by corradio** [comment]
@HansHyde keep in mind that you are comparing two things that can't be compared: you are computing **operational** emissions (fuel burn exclusively) with LCA data (where the fuel burn is not easy to identify). It's quite normal you're getting the discrepancies as you're not using the same method to "validate", as one year of LCA emissions will never give the same as one year of fuel burn emissions.
I understand what you're trying to do (i.e. take into account installed capacity), but I have a very hard time understanding how you will be able to do that with the LCA methodology (without redoing a whole LCA analysis).

I'm just saying this to prevent you from going down a rabbit hole from which we won't be able to draw anything meaningful for this project.

**Comment by brunolajoie** [comment]
Hello Hans, 
Could you explain what you meant by OCA and OA please?
Thanks!

**Comment by HansHyde** [comment]
@brunolajoie, Oli was correct in catching my repeated typo error. "OCA" means LCA.

@corradio, the California data set I provided a link to gave fuel/plant type, fuel content, and Heat Rate (even if for the entire year and more a matter of individual unit capacity factors) for every generating unit in the state, I put it all in an Excel spreadsheet and calculated the carbon dioxide that would be released from methane (gas) and summed it per fuel type. This gave me the OA value.

Then I took the number of MHr reported, summed them and multiplied the total MWH by the LCA value for gas.

OA calculation - 47,942,155.158 Metric Tonnes of CO2 would be released if calculated from 53.07 kg of CO2 per MMBTU.

LCA calculation - 51,888,731.946 Metric Tonnes of CO2 would be if calculated from 490gCO2eq/kWh using the LCA conversion.

This leaves 7.6% for everything else that would be included within the LCA.

I have also calculated a value for extraction, processing & delivering methane to a power plant (well to wheels) before it is burned to produce MWh.  This is a "Carbon Cost" of 22 kg per 1 MMBTU of natural gas.

Hence, we are looking at 75 kg of CO2 released per 1 MMBTU consumed.  This increases the "Fuel Burn" OA value above by 1.41x to 67,590,000 Metric Tonnes of CO2 would be released.

Compare that to the LCA number.

The LCA value is far too low.  But that doesn't just mean change the LCA value for gas. 

The LCA value is sadly lacking, as it is way too simplistic & it is attempting to compare apples, African Water Buffalos and prices of ice for Eskimos under one conversion.

Not picking on nuclear, but does the LCA for nuclear include 10 years to build and the 60 years it sits in SAFSTOR producing 0 kWhr of electricity, against 40 years in operation at 85% capacity factor?  This I do not know, but I know this is the actualized reality of nuclear reactors born out by the numbers looking at all reactors.  Sum the numbers together, and capacity factor dropped to roughly 30-35% over 110 years.  No different that a 20 year lifecycle of a wind turbine x 5.5 replacements.

Installed Capacity could give us a background g CO2 eq for every every hour of the year, not per kWh of operation.  The plants all exist, regardless if they are producing or not producing.  The "background" CO2 is our costs of electricity whether want to use it or not, regardless of fuel/plant type, which is more accurate than per kWh produced.

We could have a hybrid model and IMHO it would be far more accurate.  Have a good evening.

**Comment by corradio** [comment]
Hi @HansHyde ,

Thanks for the analysis.
I'm actually surprised that the LCA number is that "accurate" compared to the operational analysis that takes into account fuel content, heat rate from every installed unit, given that it is so simplistic. It does seems however that is is underestimated by ~30%, but it's not that surprising. I think even an LCA study of typical californian plants might correct this bias (compared to IPCC which aggregates all studies across the world). I would scout the literature if I were you, and try to find studies that are region-specific.
I'd also recommend looking at the studies IPCC uses if you want to dig down in how they actually compute it (for nuclear).

I'm not completely sure installed capacity is causing the discrepancy, but in order to investigate we would need to go towards a hybrid model, which would require significant work and would require peer-reviewing (people trust the IPCC more that they would trust our own study).

Thanks for clarifying nonetheless.
